### DISCUSSION TITLE
revert to stock from 2.11.2 to update to 2.13.

### DISCUSSION THREAD

**Discussion by mappet2011** [mongoose-os-apps/shelly-homekit#1596] [Q&A]
if i push the revert to stock button in homekit firmware of shelly 1 plus, it just switches to update page (see pic) but absolutely nothing happens. i saw the hint with the problem of missing reset in some cases but neither reset in sw nor pushing the reset button helps. 
![IMG_0871](https://github.com/user-attachments/assets/0358ecb2-4f7d-4b94-9c05-a6bdac379da1)

i would manually upload shelly firmware then - if i would find one?! any hints? where can i get this firmware, if not 1.4 an older one?

**Comment by markirb** [comment]
Oh. My bad, they seem to be missing. Will fix it later. See here if you are looking for files mabually
http://rojer.me/files/shelly/stock/1.3.3/

**Reply by markirb** [reply]
Should also work again via webif

**Reply by mappet2011** [reply]
thank you, perfect, i have the first one manually updated, worked great, now the other 20+ will follow 😁, i will give you a hint if webif doesnt work

**Reply by mappet2011** [reply]
no does not work, you update now to 1.14, check for update does not show a later version 😬, i have the latest one .., can i use ota link now?

**Reply by mappet2011** [reply]
so your „revert to stock“ updates to 1.14 and not 1.3, as you described in your changelog of 2.13.1. that confused me but is obvious the latest version according to shelly, so i just tried the:
http://a.b.c.d/ota?url=http://rojer.me/files/shelly/shelly-homekit-Shelly1.zip
now it is on 2.13.3 and works, thank you

**Reply by mappet2011** [reply]
sorry, dont want to annoy you, but i cannot manage the update with shelly 1 (no plus), also there nothing happens wenn "push to revert, also the manual flashing does not work (with zip file SHSW-1 from your 1.3.3 Link). If i upload manually it starts uploading (see pic), then it reboots but nothing happens, refresh IP will still show 2.11.2 .. Can you please check. 
PS: I do have plug-s and i3 in my reportiere as well .. ;-), thank you! 
![image](https://github.com/user-attachments/assets/f0c6d4d2-d0e3-4de6-b830-eae3b1a20c83)

**Reply by mappet2011** [reply]
after some further attempts i need to complete my comment, it works with some shellys but sometimes it doesnt. in these non working cases it updates to an shelly 1.9 firmware only and there it ends, i cant do anything, no way to flash manually another firmware, neither shelly nor homekit. could it be that some older models dont work? what can i do to get homekit firmware again? i fear to try further updates if it ends with shelly 1.9., no homekit.

**Reply by markirb** [reply]
You only ever need this process for Gen2/Plus models

**Reply by mappet2011** [reply]
ah, just saw the comment, sorry, but what do you mean? non plus can directly update to 2.13 then?

**Reply by markirb** [reply]
Exactly

**Reply by mappet2011** [reply]
yes, direct update to 2.13 works, thank you, i just have one shelly 1 with 1.9 now, any idea how i can bring it back to homekit?
